.github/workflows/ci.yml

```diff
@@ -77,6 +76,6 @@ jobs:
       - uses: goreleaser/goreleaser-action@v6
         with:
           version: latest
-          args: release --rm-dist
+          args: release --clean
```

Comment:
The GoReleaser deprecating some options in v2.
https://goreleaser.com/deprecations/#-rm-dist